model/src/main/java/org/cloudfoundry/identity/uaa/scim/ScimUser.java

```diff
@@ -842,4 +846,19 @@ public void patch(ScimUser patch) {
         }
     }
 
+    public static class JsonCustomAttributeSerializer extends JsonSerializer<LinkedHashMap<String, Object>> {
+        @Override
+        public void serialize(LinkedHashMap<String, Object> map, JsonGenerator gen,
+                SerializerProvider serializers) throws IOException {
+            gen.writeStartObject();
+            for(String e : map.keySet()) {
```

Comment:
There's some inconsistency in formatting in the repo, but should this have space between for and paren?